cloudflare/resource_cloudflare_custom_ssl_test.go

```diff
@@ -0,0 +1,104 @@
+package cloudflare
+
+import (
+	"fmt"
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"os"
+	"regexp"
+	"testing"
+)
+
+func TestAccCloudflareCustomSSL_Basic(t *testing.T) {
+	t.Parallel()
+	var customSSL cloudflare.ZoneCustomSSL
+	domain := os.Getenv("CLOUDFLARE_DOMAIN")
```

Comment:
`CLOUDFLARE_ZONE_ID` is preferred to `CLOUDFLARE_DOMAIN` for the uniqueness of it.

Reply:
Ok i was applying the pattern we currently have elsewhere. I agree zone id a unique identifier is preferable esp. since the tests create actual resources and cost $ potentially.

Reply:
Yeah, sorry about that. We've put the deprecation in place as of the [next minor release](https://github.com/terraform-providers/terraform-provider-cloudflare/blob/master/CHANGELOG.md#1170-unreleased) and I'll be swapping everything over internally once that lands.

Reply:
no problem, updated!